runtime/oti/VMHelpers.hpp

```diff
@@ -2158,10 +2159,15 @@ class VM_VMHelpers
 	}
 
 	static VMINLINE void
-	cleanupContinuationObject(J9VMThread *vmThread, J9Object *objectPtr)
+	cleanupContinuationObject(J9VMThread *vmThread, J9Object *objectPtr, bool verifyOnly = false)
 	{
 #if JAVA_SPEC_VERSION >= 19
-		vmThread->javaVM->internalVMFunctions->freeContinuation(vmThread, objectPtr);
+		if (verifyOnly) {
+			J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(vmThread, objectPtr);
+			assert(NULL == continuation);
```

Comment:
isn't there as API like Assert_VM_true ?

Reply:
we can try to move the method to ContinuationHelpers.cpp

we already attempted this move (most of Continuation specific method should not really be here in a generic VM Helpers file) and there were problems, but still, give a thought/try again

Reply:
there is a compiling setting problem to reference from subfolder oti to subfolder vm

Reply:
- we could pass the mode to freeContinuation, but I'm not very much fan of it (since it's shared with VM path of last unmount)
- since this is GC specific helper, this could (in fact, should) be moved to a GC *.cpp file

Reply:
can't find a good file to move it to, but so far best is base ObjectAccessBarrier
some others are ObjectModel, modronapi, VMInterfaceAPI...

Reply:
we should consider moving needScanStacksForContinuation and various related helpers, too, but not at as a part of this change